.github/workflows/client-release.yml

```diff
@@ -35,14 +36,27 @@ jobs:
         tar -czvf composable-${{ env.RELEASE_VERSION }}.tar.gz target/release/composable
         tar -czvf picasso_runtime.compact.wasm-${{ env.RELEASE_VERSION }}.tar.gz target/release/wbuild/picasso-runtime/picasso_runtime.compact.wasm
         tar -czvf parachain-utils-${{ env.RELEASE_VERSION }}.tar.gz target/release/parachain-utils
-        gsutil cp *.tar.gz gs://composable-binaries/community-releases/${{ env.RELEASE_VERSION }}/ 
+        gsutil cp *.tar.gz gs://composable-binaries/community-releases/${{ env.RELEASE_VERSION }}/     
 
     - name: Login to DockerHub
       uses: docker/login-action@v1 
       with:
         username: ${{ secrets.DOCKER_HUB_USERNAME }}
         password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}  
 
+    - name: GenerateMD5
+      run: |
+        sudo apt install -y ucommon-utils
+        composabledalichecksum=$(md5sum target/release/composable | cut -d ' ' -f 1) && echo "$composabledalichecksum composable" >> checksum.md5.txt
+    
+    - name: Upload Checksum
+      uses: svenstaro/upload-release-action@v2
+      with:
+        repo_token: ${{ secrets.COMPOSABLE_GITHUB_TOKEN }}
+        file: checksum.md5.txt
+        asset_name: checksum.md5.txt
+        tag: ${{ github.ref }}
+    
```

Comment:
i mentioned this on the devops sync, but we don't need this, srtool already provides the sha256 hash of the runtime